index.html

```diff
@@ -113,6 +113,31 @@ <h2>A project for first-time contributions</h2>
         <!-- ================================================ -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Taufiqurrahman</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="www.twitter.com" target="_blank">@fiqgant</a>
```

Comment:
kindly add proper profile link in `href` including `https://` and your twitter username.